### ISSUE TITLE
Interface to explicitly bin map updates.

### ISSUE THREAD

**Issue by ablakey** [openlayers/openlayers#13864]
I need to make multiple updates to the map, but current approaches cause a rather ugly flicker on the screen as they update separately.

For example, here's some code:

```typescript
    if (fitToScreen) {
      this.olMap.getView().fit(OccupancyGrid.getExtent(data));
    }

    olLayer.setSource(imageSource);
```

I'm not showing context, but I have a map, I'm fitting to the new extent of a new background image AND setting that new background image.

Even though these two calls are happening beside each other, it causes two map updates, very quickly, causing a flicker:
Frame 0:  The old  image + old extent
Frame 1: The old image + new extent
Frame 2: The new image + new extent

I believe there is some binning of updates, as suggested by the existence of render vs. renderSync.  But I think that's just waiting for the next frame and isn't a guarantee that changes made in one synchronous call are updated in one binned effort.  (Though I'm not sure how this actually works like this given JavaScript's single thread...)

**Comment by MoonE** [comment]
You are correct in assuming that `render` is called on animation frames while `renderSync` renders immediatly.
Assuming you are using a `ol/source/ImageStatic`. You need to preload the image which there seems to be no straight forward way to do so.

As a workaround, you can preload the source with an offscreen map:
https://codesandbox.io/s/preload-image-static-nfh9u0?file=/src/index.js)](https://codesandbox.io/s/preload-image-static-nfh9u0?file=/src/index.js

**Comment by M393** [comment]
#13076 is kind of related, would be nice to have
